README.md

```diff
@@ -137,9 +139,17 @@ You should now be able to use the AWS web console to get the DNS name for your l
 Example:
 
 ```
-export CLUSTER_NAME=gobstopper
-bin/build-cluster.sh $CLUSTER_NAME
-bin/get-params.sh $CLUSTER_NAME
-bin/add-alb.sh $CLUSTER_NAME
-bin/deploy-full.sh $CLUSTER_NAME
-```
\ No newline at end of file
+bin/build-cluster.sh gobstopper
+bin/get-params.sh gobstopper
+bin/add-alb.sh gobstopper
+bin/deploy-full.sh gobstopper
+```
+
+## Releasing a new version
+
+1. Decide on a new version number. We use [semantic versioning](https://semver.org/).
```

Comment:
a. Major versions (breaking changes) should be incremented when the version requires a cluster rebuild, routing changes, or other changes that are not backward compatible with previous versions
   b. Minor versions (features) should be incremented when the version includes new user-facing functionality - in general, the minor version should be incremented if any of the service versions change at the minor version level - i.e. if BLACKLIGHT_VERSION changes from v1.3.4 to v1.4.0, then the Camerata minor release version should increase.  The minor version only needs to increase by one regardless of how many component versions change
  c. Patch versions (fixes, refactors) should be incremented when there are enhancements or refactors to camerata that do not affect the deployment process (i.e. Documentation remains valid) and/or when updating the patch level of dependent applications